### PR TITLE
OCPBUGS-43498: pin libreswan version to 4.6 in rhcos extension

### PR DESCRIPTION
**Pull request by zshi-redhat**
This change installes libreswan-4.6-3.el9_0.3 via ipsec extension as a mitigation step to address the ipsec connection issue tracked in https://issues.redhat.com/browse/OCPBUGS-41823. This change can be reverted once a final fix in libreswan is ready to be consumed by openshift

JIRA: https://issues.redhat.com/browse/OCPBUGS-43498

### CODE REVIEW

File: extensions-ocp-rhel-9.4.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ extensions:
   # https://github.com/coreos/fedora-coreos-tracker/issues/1504
   ipsec:
     packages:
-      - libreswan
+      - libreswan-4.6
```
Comment (jlebon) [inline extensions-ocp-rhel-9.4.yaml:18]: Can you add a comment just above this with a link to the Jira ticket so it's easier to tell when to revert this change?

**Review by jlebon — COMMENTED** [review]
We also need this change in `extensions-ocp-rhel-9.6.yaml`. Unless it's fixed in 9.6?

File: extensions-ocp-rhel-9.4.yaml (hunk 1 above)
Comment (zshi-redhat) [reply extensions-ocp-rhel-9.4.yaml:18]: ticket for reverting the change made in this PR: https://issues.redhat.com/browse/SDN-5544

**Comment by sdodson** [comment]
/jira-clone release-4.18 release-4.17 release-4.16 release-4.15

**Comment by sdodson** [comment]
/jira backport release-4.18,release-4.17,release-4.16,release-4.15
